data/data/aws/vpc/sg-master.tf

```diff
@@ -31,9 +31,9 @@ resource "aws_security_group_rule" "master_ingress_icmp" {
   security_group_id = "${aws_security_group.master.id}"
 
   protocol    = "icmp"
-  cidr_blocks = ["${data.aws_vpc.cluster_vpc.cidr_block}"]
```

Comment:
can we keep the cidr restricted to vpc ?

Reply:
I was matching what was in sg-worker.txt. What's important is that we can receive ICMP from the nlbs. I'm not sure what the source IP will be for that.

(it also looks like the nlb creates an additional rule allowing itself - i need to investigate that)